### ISSUE TITLE
Team view

### ISSUE THREAD

**Issue by vapetrov** [uiuc-mrdc/game-server#3]
Currently pages are only visible on the internal network, and if teams want information they need to see in on the projectors or ask a committee member.
Instead, teams should be able to view an internet-facing page where they can see how many points they scored, when they will be up next, and how they are doing in the bracket.

**Comment by lpulley** [comment]
Agreed. Django appears to have a native plugin for authentication/accounts, or something like that. I'm keeping an eye out for that in the tutorials, since I think it would be nice to have the outward-facing site be the same as the internal one, just with authentication for access to internal components.
